### PR TITLE
OCPBUGS-35198#adding wildcards to links

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions:
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-35198

Link to docs preview:
https://77222--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/install_config/configuring-firewall.html#configuring-firewall_configuring-firewall

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/configuring-firewall.adoc [hunk 1]
```diff
@@ -189,15 +189,15 @@ Alternatively, if you choose to not use a wildcard for AWS APIs, you must includ
 |443
 |Used to install and manage clusters in an AWS environment.
 
-|`s3.amazonaws.com`
+|`*.s3.amazonaws.com`
 |443
 |Used to install and manage clusters in an AWS environment.
 
-|`s3.<aws_region>.amazonaws.com`
+|`*s3.<aws_region>.amazonaws.com`
```
Comment (ShaunaDiaz) [inline modules/configuring-firewall.adoc:196]:
```suggestion
|`*.s3.<aws_region>.amazonaws.com`
```

File: modules/configuring-firewall.adoc [hunk 2]
```diff
@@ -189,15 +189,15 @@ Alternatively, if you choose to not use a wildcard for AWS APIs, you must includ
...
 |443
 |Used to install and manage clusters in an AWS environment.
 
-|`s3.dualstack.<aws_region>.amazonaws.com`
+|`*s3.dualstack.<aws_region>.amazonaws.com`
```
Comment (ShaunaDiaz) [inline modules/configuring-firewall.adoc:200]:
```suggestion
|`*.s3.dualstack.<aws_region>.amazonaws.com`
```
I believe that with-the-period is correct. Best-practice to have QE look before peer review, IMHO.

**Review by ShaunaDiaz — COMMENTED** [review]
Confirm the proper syntax for the wildcard with QE and update the examples as needed.

File: modules/configuring-firewall.adoc (hunk 2 above)
Comment (gpei) [reply modules/configuring-firewall.adoc:200]: yes, it supposed to be `*.s3.dualstack.<aws_region>.amazonaws.com`

Comment (brendan-daly-red-hat) [reply modules/configuring-firewall.adoc]: @gpei, Thanks for your comment. Can you review the updated wildcard syntax?

**Pull request merged**